src/utils/helpers.ts

```diff
@@ -14,8 +14,17 @@ const truncateHash = (revision: RevisionsList['revision']) =>
 
 // return only most recent commit message
 // first commit is usually 'try_task_config'
-const getLatestCommitMessage = (item: RevisionsList) =>
-  item.revisions[item.revisions.length - 1].comments;
+const getLatestCommitMessage = (item: RevisionsList) => {
```

Comment:
@julienw Is this all you had in mind regarding this issue?

Reply:
Yes!
Thanks, this looks pretty good to me!

I think that in the future we could display the commit count in the item too. Out of scope for this patch (we'd probably need to update the design a bit).